src/components/Filters/CollapsibleFilter/CollapsibleFilter.js

```diff
@@ -0,0 +1,55 @@
+import './CollapsibleFilter.less';
+import getIcon from '../../iconMap';
+import PropTypes from 'prop-types';
+import { useState } from 'react';
+
+const CollapsibleFilter = ({
+  hasChildren = true,
+  className = '',
+  title,
+  desc,
+  children,
+}) => {
+  const [isOpen, setOpen] = useState(hasChildren);
+
+  const opened = (
+    <span className="o-expandable__cue-close" role="img" aria-label="Hide">
+      {getIcon('minus-round')}
+    </span>
+  );
+  const closed = (
+    <span className="o-expandable__cue-open" role="img" aria-label="Show">
+      {getIcon('plus-round')}
+    </span>
+  );
+
+  return (
+    <section className={`o-expandable ${className}`}>
+      <button
+        className="o-expandable__header"
+        aria-expanded={isOpen}
+        aria-label={`Hide ${title} filter`}
+        onClick={() => setOpen(!isOpen)}
+      >
+        <h3 className="o-expandable__label">{title}</h3>
+        <span className="o-expandable__cues">{isOpen ? opened : closed}</span>
+      </button>
+      {isOpen ? (
+        <div className="o-expandable__content">
+          <p>{desc}</p>
+          {children}
+        </div>
+      ) : null}
+    </section>
+  );
+};
+
+CollapsibleFilter.propTypes = {
+  hasChildren: PropTypes.bool,
+  className: PropTypes.string,
+  title: PropTypes.string,
+  desc: PropTypes.string,
+  children: PropTypes.node,
+};
```

Comment:
can you add the .isRequired to these?  

```
 hasChildren: PropTypes.bool.isRequired,
  className: PropTypes.string.isRequired,
  title: PropTypes.string.isRequired,
  desc: PropTypes.string.isRequired,
  children: PropTypes.node.isRequired,
```

Can you also remove the default value from hasChildren ?  I think we should explicitly declare the value when we're using Collapsible filter. 

This will help us catch errors more easily.

In <FederalState, hasChildren is passed in as a prop, but in FilterPanel, <FederalState is used without any props.

Reply:
This has been updated

Reply:
Updated again to fix unit tests. Also, is className required? Based on the logic within the component that checks for its existence, it doesn't appear that it is. I've removed the `required` attribute for it, based on the previous implementation

Reply:
You're right. className should be optional